src/transformers/models/tapas/tokenization_tapas.py

```diff
@@ -1969,11 +1969,11 @@ def convert_logits_to_predictions(self, data, logits, logits_agg=None, cell_clas
             answer_coordinates = sorted(answer_coordinates)
             predicted_answer_coordinates.append(answer_coordinates)
 
-        output = predicted_answer_coordinates
+        output = (predicted_answer_coordinates,)
```

Comment:
The documentation of this method says:

```
:obj:`tuple` comprising various elements depending on the inputs
```

but that is currently not true, as if `logits_agg` is not passed, it will simply return `predicted_answer_coordinates`. I think it would be more coherent to do like it is done in the documentation, that is, to only output tuples (which is similar to what we do with model outputs).

Reply:
Please note that this change breaks your scripts @NielsRogge, as where you would usually do
```py
answer_coordinates = convert_logits_to_predictions(...)
```
you would now need to do:
```py
answer_coordinates = convert_logits_to_predictions(...)[0]
```
or
```py
answer_coordinates, = convert_logits_to_predictions(...)
```
I'm open for discussion if you disagree with this change.

Reply:
For me this is fine 👍